src/App.js

```diff
@@ -70,35 +73,47 @@ class App {
 				// Success handler.
 				() => {
 					// Insert modified HTML.
-					$( '.mw-parser-output' ).html( self.api.getReplacementHtml() );
-					// Highlight when hover a user's contributions.
-					// @TODO This is just testing code and should be
-					// replaced by the proper behaviour.
-					$( '.mw-parser-output .editor-token' )
-						.on( 'mouseenter', function () {
-							const ids = self.api.getIdsFromElement( this );
-							// Activate all this user's contribution spans.
-							$( '.token-editor-' + ids.editorId ).addClass( 'active' );
-							// Information for popup.
-							// eslint-disable-next-line no-console
-							console.log( self.api.getTokenInfo( ids.tokenId ) );
-						} )
-						.on( 'mouseleave', function () {
-							// Deactivate all spans.
-							$( '.mw-parser-output .editor-token' ).removeClass( 'active' );
-						} );
-					self.widget.setState( 'ready' );
+					$( '.mw-parser-output' ).html( this.api.getReplacementHtml() );
+					this.attachContentListeners();
+					this.widget.setState( 'ready' );
 				},
 				// Error handler.
 				errorCode => {
-					self.widget.setState( 'err' );
-					self.widget.setErrorMessage( errorCode );
+					this.widget.setState( 'err' );
+					this.widget.setErrorMessage( errorCode );
 				}
 			);
 	}
 
 	/**
-	 * Repond to the close event that the widget emits.
+	 * Add listener to highlight attribution and show the RevisionPopupWidget.
+	 */
+	attachContentListeners() {
+		$( '.mw-parser-output .editor-token' )
+			.on( 'mouseenter', e => {
+				const ids = this.api.getIdsFromElement( e.target );
+				// Activate all this user's contribution spans.
+				$( '.token-editor-' + ids.editorId ).addClass( 'active' );
+			} )
+			.on( 'mouseleave', () => {
+				// Deactivate all spans.
+				$( '.mw-parser-output .editor-token' ).removeClass( 'active' );
+			} );
+
+		// Create RevisionPopupWidget instance.
+		const revisionPopup = new RevisionPopupWidget();
+		$( document.body ).append( revisionPopup.$element );
```

Comment:
why not just `$( 'body' )` ?

Reply:
I got that from you! ;-) https://app.slack.com/client/T024KLHS4/CDG6XHT3Q

`'body'` works just as well, I shall change it.

Reply:
hah!

I am actually not 100% sure if there's a good difference. I'm just more used to (and I think we use it elsewhere in WWT) to the `$('body')` version. I should look into the difference.